neo4j/neo4j.adoc

```diff
@@ -0,0 +1,28 @@
+= How To Use Neo4j
+:section: Neo4j
+:section-link: neo4j
+:section-level: 1
+:slug: neo4j
+
+= image:{img}icon-guide-ecosystem.png[] How To Use Neo4j
+
+There are a variety of ways to interact and use Neo4j.
+This guide introduces some of the different products at Neo4j that suit different business and technology needs and how to get started using them.
+
+From our core graph database to visualization for business users, we have a variety of opportunities to maximize your business and simplify your data.
```

Comment:
edit: "From our core **product, the** graph database**,** to visualization for business users, we have a variety of ~~opportunities~~ **offerings that** ~~to~~ maximize your business and simplify your data."